drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/util/KogitoDMNService.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.kogito.client.util;
+
+import org.drools.workbench.screens.scenariosimulation.model.typedescriptor.FactModelTuple;
+import org.jboss.errai.bus.server.annotations.Remote;
+import org.jboss.errai.common.client.api.ErrorCallback;
+import org.jboss.errai.common.client.api.RemoteCallback;
+import org.kie.workbench.common.dmn.webapp.kogito.marshaller.js.model.dmn12.JSITDefinitions;
+import org.uberfire.backend.vfs.Path;
+
+/**
+ * Interface required because <b>runtime</b> and <b>testing</b> environments would
+ * need/provide different implementations
+ */
+@Remote
```

Comment:
Are you sure about this annotation? I'm not an `Errai`expert but I guess this should be used only to for remote invocation. Can you please verify?

Reply:
@danielezonca in my opinion, it's not necessary. Easy reason: we don't have a server. Other reasons: this remote should be put in `api` module to have sense (it should be shared between `client` and `server` according to ERRAI doc). Implementing services should have @Service annotation, and in this case is missing. I'll remove it, in case of unexpected error I can easily restore it.